pkg/resources/view_acceptance_test.go

```diff
@@ -1,109 +1,282 @@
+					resource.TestCheckResourceAttr("snowflake_view.test", "name", viewName),
+					resource.TestCheckResourceAttr("snowflake_view.test", "tag.#", "1"),
+					resource.TestCheckResourceAttr("snowflake_view.test", "tag.0.name", "tag2"),
+					resource.TestCheckResourceAttr("snowflake_view.test", "tag.0.value", "some_value"),
 				),
 			},
+			// IMPORT
+			{
+				ConfigVariables:         m(),
+				ResourceName:            "snowflake_view.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"or_replace", "tag"},
+			},
 		},
 	})
 }
 
-func TestAcc_ViewWithCopyGrants(t *testing.T) {
-	accName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+func TestAcc_View_Rename(t *testing.T) {
+	viewName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	newViewName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	query := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"
 
-	resource.ParallelTest(t, resource.TestCase{
-		Providers:    acc.TestAccProviders(),
-		PreCheck:     func() { acc.TestAccPreCheck(t) },
-		CheckDestroy: nil,
+	m := func() map[string]config.Variable {
+		return map[string]config.Variable{
+			"name":        config.StringVariable(viewName),
+			"database":    config.StringVariable(acc.TestDatabaseName),
+			"schema":      config.StringVariable(acc.TestSchemaName),
+			"comment":     config.StringVariable("Terraform test resource"),
+			"is_secure":   config.BoolVariable(true),
+			"or_replace":  config.BoolVariable(false),
+			"copy_grants": config.BoolVariable(false),
+			"statement":   config.StringVariable(query),
+		}
+	}
+	m2 := m()
+	m2["name"] = config.StringVariable(newViewName)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: testAccCheckViewDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: viewConfig(accName, true, "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES", acc.TestDatabaseName, acc.TestSchemaName),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_View_basic"),
+				ConfigVariables: m(),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("snowflake_view.test", "name", accName),
-					resource.TestCheckResourceAttr("snowflake_view.test", "database", acc.TestDatabaseName),
-					resource.TestCheckResourceAttr("snowflake_view.test", "comment", "Terraform test resource"),
-					resource.TestCheckResourceAttr("snowflake_view.test", "copy_grants", "true"),
-					checkBool("snowflake_view.test", "is_secure", true), // this is from user_acceptance_test.go
+					resource.TestCheckResourceAttr("snowflake_view.test", "name", viewName),
```

Comment:
same case as with the materialized views, maybe add other simple params (like comment) to see if rename won't break other changes?

Reply:
check above